src/AuthenticatedAPIClient/tests/AuthenticatedAPIClient.test.jsx

```diff
@@ -133,6 +134,10 @@ describe('AuthenticatedAPIClient auth interface', () => {
     expect(client.get).toHaveBeenCalled();
   });
 
+  it('has method handleRefreshAccessTokenFailure', () => {
```

Comment:
1. Kill this? :)  Isn't it good enough to just be testing this method?
2. Maybe rename 'has method getCsrfToken' above too?

Reply:
Makes Sense!